src/main/java/zoe/Task.java

```diff
@@ -0,0 +1,64 @@
+package zoe;
+
+/**
+ * Superclass for all forms of tasks
+ * Contains some global methods so that all subclasses have access to that method
+ */
+public class Task {
+    protected String description;
+    protected String type;
```

Comment:
Is it better to have these attributes final? As I think they can be confirmed upon initialization.